docs/UserGuide.md

```diff
@@ -50,55 +56,63 @@ WorkBook (WB) is a **desktop app for CS/tech students who are applying for inter
+| **Find**   | `find c/COMPANY`                                                                                                                                                                                      | r/ROLE | s/STAGE​​`<br> e.g.,`find c/Meta` 
+| **Delete** | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                                                   |
+| **List**   | `list`                                                                                                                                                                                                |
+| **Undo**   | `undo`                                                                                                                                                                                                |
+| **Redo**   | `redo`                                                                                                                                                                                                |
+| **Clear**  | `clear`                                                                                                                                                                                               |
+| **Help**   | `help`                                                                                                                                                                                                |
+| **Exit**   | `exit`                                                                                                                                                                                                |
 
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:**<br>
 
+* Commands without examples are considered trivial and can be executed by inputting the command without additional parameters.
+* Commands are **case-sensitive**!
+
+</div>
 
-### Adding an internship application: `add`
+### 4.1. Adding an internship application: `add`
 
 Adds an internship application to WorkBook.
 
 Format: `add c/COMPANY r/ROLE s/STAGE [d/DATE] [e/COMPANY_EMAIL] [l/LANGUAGE TAG] [t/TAG]…​`
 
 * Adds an internship application to the list in sorted order.
-* Fields not square-bracketed are mandatory.
-* Date must be properly formatted (DD-MMM-YYYY HH:mm) when inputted.
-* Date can represent the date it happened or the deadline of the corresponding `stage`
-* Email domain, if included, must be properly formatted (example@domain.com).
-* An application can have any number of tags and language tags and are added one by one (i.e.
-if you want 2 tags, you have to specify `t/two t/tags`).
+  * Note: Internship applications **in the past** are placed at the bottom.
```

Comment:
I think this is better in the "things to note" section. Could go into a bit more detail on the sorting order enforced